site/content/how-to-monetize-apps/monetization.md

```diff
@@ -0,0 +1,92 @@
+---
+weight: 10 
+---
+
+# How to monetize apps?
```

Comment:
I'd consider keeping `Introduction` as a first header. The current docs layout has the document title already in the left sidebar.

Reply:
What about the text below? Should it be deleted or merged with the Introduction section?

Reply:
How about: ![image](https://user-images.githubusercontent.com/42515529/57523489-68c16600-7325-11e9-9065-66a0163ca9d6.png)

Reply:
Nice! How about adding an additional one-liner at the beginning of "If you're new...":
>LiveChat is an online customer service platform.
This way we can smuggle a wider context.